src/styles/_catalog.scss

```diff
@@ -1,10 +1,27 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  padding: 20px;
 
   max-width: 944px;
   margin: 0 auto;
 
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, 1fr);
+  }
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
Gaps between cards are different depending on page size, if i understand task correctly they should not. It may be becuase of using 1fr instead of 200px in column size, but i'm not sure.